Index: helloworld - 副本 (2).txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>这是副本2\r\n先提交\r\n再提交一次
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/helloworld - 副本 (2).txt b/helloworld - 副本 (2).txt
--- a/helloworld - 副本 (2).txt	(revision 9a511823b5dd74e9cde7625b30a4f39d51300b4c)
+++ b/helloworld - 副本 (2).txt	(date 1648051874859)
@@ -1,3 +1,2 @@
 这是副本2
-先提交
-再提交一次
\ No newline at end of file
+先提交
\ No newline at end of file
